android/clientmanager/src/main/java/io/mosip/registration/clientmanager/entity/UserPassword.java

```diff
@@ -0,0 +1,56 @@
+package io.mosip.registration.clientmanager.entity;
+
+import androidx.annotation.NonNull;
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+import java.io.Serializable;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity(tableName = "user_pwd")
+@Data
+@EqualsAndHashCode(callSuper=false)
+public class UserPassword implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @NonNull
+    @PrimaryKey
+    @ColumnInfo(name = "usr_id")
+    private String usrId;
+
+    @ColumnInfo(name = "pwd")
+    private String pwd;
+
+    @ColumnInfo(name = "pwd_expiry_dtimes")
+    private String pwdExpiryDtimes;
+
+    @ColumnInfo(name = "status_code")
+    private String statusCode;
+
+    @ColumnInfo(name = "lang_code")
```

Comment:
langCode column is not required, lets remove it.

Reply:
Removed langCode column from UserPassword table